PrestaShop/ps_linklist
# Uninstall module used on PS 1.6 before using this one

## How to test
1. On a 1.7.7.x shop, be sur that both `blockcms` & `ps_linklist` are not installed
2. Install [this **modified** version of `blockcms`](https://github.com/PrestaShop/ps_linklist/files/5886684/blockcms.zip) (only version compatibility range has been modified to be installable on a 1.7 shop)
3. Try to install `ps_linklist` with this PR
4. `blockcms` should be uninstalled and `ps_linklist` should be installed
